packages/ember-debug/lib/handlers.js

```diff
@@ -0,0 +1,27 @@
+import isPlainFunction from 'ember-debug/is-plain-function';
+
+export let HANDLERS = { };
```

Comment:
Ah never mind I thought for a second this was going to hold a map of arrays. This is clearly not the case https://github.com/emberjs/ember.js/pull/11833/files#diff-889b83df6e6038c39617a4a40aec777fR12.